### PR TITLE
Shifting IntersectionObserver and AudioContext to NPM

### PR DESCRIPTION
**Pull request by prateekbh**
- Shifts these deps to NPM instead of polyfill.io 
Polyfill.io is stalling paint for a long time. See here: https://www.webpagetest.org/result/200625_H5_3c838dde8c4c32e895d064ccf7ce10dc/1/details/#waterfall_view_step1

### CODE REVIEW

**Comment by prateekbh** [comment]
Observing perf improvements: https://www.webpagetest.org/result/200706_H5_9ab5d3d147968bb76411d370e99cb0e9/1/details/#waterfall_view_step1

File: pages/_app.tsx [hunk 1]
```diff
@@ -4,6 +4,12 @@ import Head from "next/head";
 import "@styles/index.css";
 import NProgress from "nprogress";
 
+if (process.browser) {
```
Comment (ritz078) [inline pages/_app.tsx]:
```suggestion
if (IN_BROWSER) {
```

**Review by ritz078 — APPROVED** [review]
LGTM

**Comment by ritz078** [comment]
@allcontributors please add @prateekbh for code.

**Pull request merged**